core/tests/musig2.rs

```diff
@@ -159,9 +159,8 @@ async fn test_musig2_script_spend() {
     let agg_xonly_pubkey =
         bitcoin::XOnlyPublicKey::from_slice(&musig_agg_xonly_pubkey.serialize()).unwrap();
     let musig2_script = bitcoin::script::Builder::new()
-        .push_int(1)
         .push_x_only_key(&agg_xonly_pubkey)
-        .push_opcode(OP_CHECKSIGVERIFY)
+        .push_opcode(OP_CHECKSIG)
```

Comment:
The op `OP_CHECKSIGVERIFY agg_xonly_pubkey` is equivalent to `OP_CHECKSIG 1 agg_xonly_pubkey`?

Reply:
If this is a question, then no. OP_CHECKSIGVERIFY does pop the stack element 1 after the verification. I did this because for some reason the test did not pass in its previous form.

Reply:
a script should end with a TRUE element after executing so, op_checksigverify pops the true

so
op_checksig

is equal to

op_checksigverify
1